nfabackend/src/main/java/com/msg/nfabackend/entities/Project.java

```diff
@@ -2,29 +2,38 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
+import javax.xml.bind.annotation.XmlType;
 
+/**
+ * Entity-Mapping for the table 'nfaprojekt'.
+ * 
+ * @author <a href="mailto:alla.bors@fau.de">Alla Bors</a>
+ */
 @Entity
 @Table(name="nfaprojekt")
```

Comment:
I would suggest an "all english" implementation. Currently you are switching between two languages.
(nfaprojekt vs. Project)
This is confusing at best and for some people in our team rather unhandy, since they need to understand everything as well.

Reply:
I agree. But I used class developed from Harish

Reply:
Yes, I made that "Project" name. I was not aware of language during initial setup. Lets try to change in next sprint.